federated_learning/breast_density_challenge/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# use python base image
```

Comment:
this doesn't seem to include a cuda/pytorch base, is it expected?

Reply:
We used the nvidia pytorch base image first but the container image size ended up being >15 GB. Same with the MONAI base image. Using just the Python base image like this, the container size is reduced to just 4 GB. That's desireable for the challenge platform.